README.md

```diff
@@ -1,149 +1,76 @@
 [![Build Status](https://travis-ci.org/IBM/watson-banking-chatbot.svg?branch=master)](https://travis-ci.org/IBM/watson-banking-chatbot)
 ![Bluemix Deployments](https://deployment-tracker.mybluemix.net/stats/3999122db8b59f04eecad8d229814d83/badge.svg)
 
-# Multi - Domain Enterprise Digital Assistant
-
-In this developer journey we will create a Watson Conversation, NLU & RnR based chatbot for multiple domains.
-that provides users with:
-
-* Answers important FAQs queries from the multiple domains such as across business lines of Loans, Account Opening, Market Place & Digital Banking of Banking industry.
-* Also the chatbot solves transactional queries such as view transactions, balances, pay utility bills etc.
+# Create a banking chatbot with FAQ discovery, anger detection and natural language understanding
+In this developer journey, we will create a chatbot using Node.js and Watson Conversation. The Conversation flow will be enhanced by using Natural Language Understanding to identify entities and using Tone Analyzer to detect customer emotions. For FAQs, a call to the Discovery service will use passage retrieval to pull answers from a collection of documents.
 
 When the reader has completed this journey, they will understand how to:
 
-* Create a chatbot with dialogs using Watson Conversation on NodeJS platform
-* Learn how to best create Retrieve n Rank collection for long tail questions.
-* Get enhanced natural language capability through NLU API
-* Understand the Banking domain for creating a chatbot & using Conversation and Retrieve n Rank in sync.
-
-
-## Technical Architecture - Enterprise Digital Assistant
+* Create a chatbot that converses via a web UI using Watson Conversation and Node.js
+* Use Watson Discovery with passage retrieval to find answers in FAQ documents
+* Use Watson Tone Analyzer to detect emotion
+* Identify entities with Watson Natural Language Understanding
 
 ![](doc/source/images/Banking-Architecture.png)
 
+### With Watson
 
-## Flow
```

Comment:
we want to keep the flow, but we'll need an architecture diagram, i'm not sure the existing one accurately represents what this app does. i'm happy to help make one if someone provides me with a sketch.